templates/compositions/ec-component-navigation-links/navigation-links.html.twig

```diff
@@ -0,0 +1,69 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for ECL Navigation Links.
+   */
+    Parameters:
+      - "title" (string) (default: ''): title of the child
+      - "href" (string) (default: ''): url of the child
+      - "description" (string) (default: ''): short description
+      - "links" (array) (default: []): collection of standalone @ecl-twig/link items
+      - "extra_classes" (string) (default: '')
+      - "extra_attributes" (array) (default: []): format: [
+          {
+            "name" (string) (default: ''),
+            "value" (string) (default: '')
+          },
+          ...
+        ]
+#}
+
+{# Internal properties #}
+
+{% set _title = title|default('') %}
+{% set _href = href|default('') %}
+{% set _description = description|default('') %}
+{% set _links = links|default([]) %}
+{% set _css_class = '' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+<div class="{{ _css_class }}" {{ _extra_attributes|raw }}>
+  <h2 class="ecl-u-ma-none ecl-u-type-prolonged-xl ecl-u-type-bold">
+    {% include '@ecl-twig/link' with {
+      'link': {
+        'type': 'standalone',
+        'label': _title,
+        'path': _href,
+      }
+    } %}
+  </h2>
+  {% if _description is not empty %}
```

Comment:
`A summary should always be used in the absence of links`
Looks like we should have the additional condition. So we should show this summary if we don't have links or not iterative.

Reply:
We can use a summary and have links as well, so its not an strict relationship. I don't think we need to force that logic here.

Reply:
Ok for me.